doc/nrf/libraries/networking/nrf_cloud_alerts.rst

```diff
@@ -1,7 +1,7 @@
-.. _lib_nrf_cloud_alerts:
+.. _lib_nrf_cloud_alert:
 
-nRF Cloud Alerts
-################
+nRF Cloud Alert
```

Comment:
Search for "alerts" in this file and check whether some of the should be replaced with Alert/alert. At least on lines 11 and 12, there are occurrences that I'd change.